deploy/crds/pulpproject_v1beta1_pulp_cr.ci.yaml

```diff
@@ -6,6 +6,8 @@ spec:
   # image: pulp
   tag: "latest"
   admin_password_secret: "example-pulp-admin-password"
+  pulp_settings:
+    content_origin: http://localhost:24880
```

Comment:
I don't understand why, but `http://$(minikube ip):24880/pulp/content/` fails
Example:
http://192.168.49.2:24880/pulp/content/zbrrs/
https://github.com/pulp/pulp-operator/runs/2623683331?check_suite_focus=true#step:11:828

But declaring 
```
content_origin: http://localhost:24880
```
works!

Reply:
is it because of the port-forwarding?

Reply:
actually I didn't try without port-forwarding, let me check!

Reply:
I removed the port-forwarding and it failed =/
https://github.com/pulp/pulp-operator/runs/2624811899?check_suite_focus=true#step:11:807

Reply:
Yeah, don't think your original update was wrong just a side-effect of how we are running things in minikube. Was just giving context as to why it would pass with the above setting.

Reply:
I thought you were helping me to debug!
I changed back to the original approach